<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetFirewall</class>
 <widget class="QWidget" name="widgetFirewall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firewall</string>
  </property>
  <property name="windowIcon">
   <iconset resource="firewall.qrc">
    <normaloff>:/icons/firewall48.png</normaloff>:/icons/firewall48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupFirewall">
     <property name="title">
      <string>Enable Firewall</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushEdit">
          <property name="text">
           <string>Change Firewall Rules</string>
          </property>
          <property name="icon">
           <iconset resource="firewall.qrc">
            <normaloff>:/icons/firewall48.png</normaloff>:/icons/firewall48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushReset">
          <property name="text">
           <string>Revert Firewall Rules</string>
          </property>
          <property name="icon">
           <iconset resource="firewall.qrc">
            <normaloff>:/icons/firewall48.png</normaloff>:/icons/firewall48.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>642</width>
          <height>427</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFirewall">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="firewall.qrc"/>
 </resources>
 <connections/>
</ui>
